include/mbedtls/ssl.h

```diff
@@ -1811,6 +1811,10 @@ struct mbedtls_ssl_context {
 
     size_t MBEDTLS_PRIVATE(in_hslen);            /*!< current handshake message length,
                                                     including the handshake header   */
```

Comment:
Please add some tests, i.e. at least one test case in `tests/ssl-opt.sh` that fails before adding this feature and passes after adding it. This will probably require adding things to `programs/ssl/ssl_{client,server}2.c`.